backend/persistent/PlotlyAPI.js

```diff
@@ -38,7 +38,7 @@ export function newDatacache(payloadJSON, type) {
     }).then(res => {
         return res;
     }).catch(err => {
-        console.error(err);
+        console.error(err); // eslint-disable-line no-console
```

Comment:
Let's actually replace this with a `Logger.log` call (same goes for the rest of the backend error handling)

Reply:
Alright, I'll replace 'em.